### PR TITLE
Docu 3303 - Site Refresh Epic branch

### PR DESCRIPTION
**Pull request by fabianrbz**
### Description

What did you change and why?
 
[Jira Epic](https://konghq.atlassian.net/browse/DOCU-3303)

Includes the following tickets:
* [DOCU-3304](https://konghq.atlassian.net/browse/DOCU-3304): Font updates
* [DOCU-3305](https://konghq.atlassian.net/browse/DOCU-3305): Left side navigation design refresh
* [DOCU-3306](https://konghq.atlassian.net/browse/DOCU-3306): Plugin Hub nav item submenu
* [DOCU-3348](https://konghq.atlassian.net/browse/DOCU-3348): Move search into top nav
* [DOCU-3361](https://konghq.atlassian.net/browse/DOCU-3361): Apply "blue page" styles to docs pages
* [DOCU-3367](https://konghq.atlassian.net/browse/DOCU-3367): Docs nav item submenu

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

[DOCU-3304]: https://konghq.atlassian.net/browse/DOCU-3304?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOCU-3305]: https://konghq.atlassian.net/browse/DOCU-3305?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOCU-3306]: https://konghq.atlassian.net/browse/DOCU-3306?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOCU-3348]: https://konghq.atlassian.net/browse/DOCU-3348?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOCU-3361]: https://konghq.atlassian.net/browse/DOCU-3361?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOCU-3367]: https://konghq.atlassian.net/browse/DOCU-3367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by fabianrbz** [comment]
arrow icons added 
Screenshot 2023-08-15 at 09 29 31

**Comment by acgoldsmith** [comment]
@fabianrbz , I can't open the Docs or Plugin Hub mega menus either by clicking name or arrow

File: app/_data/landing_page.yml [hunk 1]
```diff
@@ -3,98 +3,106 @@ features:
...
+            icon: /assets/images/icons/icn-doc.svg
+            url: /konnect/dev-portal/
           - text: Self-managed
-            icon: /assets/images/icons/icn-server-blue.svg
+            icon: /assets/images/icons/icn-doc.svg
             url: /gateway/latest/kong-enterprise/dev-portal/
 
-      - name: Vitals
+      - name: Analytics
         description: View granular analytics with Vitals to monitor the health of environments and clusters. Gain deep insights into service, route, and application usage.
```
Comment (acgoldsmith) [inline app/_data/landing_page.yml:24]:
```suggestion
        description: View granular analytics with Analytics to monitor the health of environments and clusters. Gain deep insights into service, route, and application usage.
```

**Comment by fabianrbz** [comment]
@acgoldsmith Fixed. Sorry about that, it broke when I pushed the a11y changes.

**Comment by Guaris** [comment]
I know I complained about this being centered before, and now it's left-aligned and I think center would be better. Ideally with the icon of topics aligning with the icon for the section. 
image.



The URL link-out icon here would be better beside the topic otherwise it looks like clicking the card will link me elsewhere. 
image


The word DOCS feels out of place, and i think it might be creating too much white space in this drop down

image


I don't feel very strong about this but in this image, the link-out icon buttons feel like they are floating in space. The plugin hub version looks much better with the icons for each product. 

image

Does this home icon feel just a touch too small to anyone else?

image


On the old website the white background hides that this icon looks grainy on my browser, but on the blue it looks a bit fuzzy
image
If I zoom in from 100% to 200% I can tell it isn't grainy
image

In Deck the FAQ icon has a lot of whitespace at the top, 
image
the old one with the little square seems a better fit
image


Mine were just small nit picks, these changes are great.

**Comment by cloudjumpercat** [comment]
> The word DOCS feels out of place, and i think it might be creating too much white space in this drop down
I felt the same way. Maybe if we add more "sections" to our docs we can revisit this and add "Docs" in down the road, but for now I think it feels out of place/self explanatory.

> Does this home icon feel just a touch too small to anyone else?
I think the home icon felt like the correct size to me.

Overall, I really like the look and feel of the redesign. I think the dropdown arrows on "Docs" and "Plugin Hub" at the top menu bar are great.

**Comment by fabianrbz** [comment]
> I know I complained about this being centered before, and now it's left-aligned and I think center would be better. Ideally with the icon of topics aligning with the icon for the section.

Yeah, unfortunately, that's not so easy to implement and might look too unbalanced.

> The URL link-out icon here would be better beside the topic otherwise it looks like clicking the card will link me elsewhere.

Fixed

> The word DOCS feels out of place, and i think it might be creating too much white space in this drop down
> I felt the same way. Maybe if we add more "sections" to our docs we can revisit this and add "Docs" in down the road, but for now I think it feels out of place/self explanatory.

I'll drop this

> Does this home icon feel just a touch too small to anyone else?
I think the home icon felt like the correct size to me.

I'll bump it 2pxs so that it matches the icons on the right

> On the old website the white background hides that this icon looks grainy on my browser, but on the blue it looks a bit fuzzy

Which browser are you using? and which resolution? They don't look grainy at all on my machine (Chrome 116.0.5845.96), they are svgs so they should scale nicely.

> In Deck the FAQ icon has a lot of whitespace at the top,

I don't remember changing this, maybe @lena-larionova updated it when she worked on the sidenav so that it matches the new designs?

> I don't feel very strong about this but in this image, the link-out icon buttons feel like they are floating in space. The plugin hub version looks much better with the icons for each product.

Yeah, we don't have icons for all of them, so we decided to leave it as is 🤷 I'll see what I can do about the link-out

**Comment by acgoldsmith** [comment]
@fabianrbz , let's make some additional tweaks, please:
* remove the grey from page behind overlay
* narrow down the size of the overlay window so it's about the same width as the left nav
* when I click the close X  there's a second step I have to click for "skip to content" - is it possible to remove this? 
Screenshot 2023-08-24 at 8 41 32 AM

**Comment by fabianrbz** [comment]
@acgoldsmith I think that comment applies to https://github.com/Kong/docs.konghq.com/pull/5901

**Comment by acgoldsmith** [comment]
> @acgoldsmith I think that comment applies to #5901

It sure does :) I'll move

**Review by cloudjumpercat — APPROVED** [review]

**Review by acgoldsmith — APPROVED** [review]
LGTM! Thank you

**Pull request merged**